helm/files/create_indexd_user

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Might make sense to completely remove these scripts as I do not expect anyone executing them from a pod, then conditionally executing the create user command within say the wsgi/local_settings.py file. The migration script I think should be executed manually on an as needed basis

Reply:
Thanks, I have removed the scripts from helm charts.